src/style/App.css

```diff
@@ -0,0 +1,6 @@
+#App{
+    margin: auto;
+    max-width: 700px;
+    display: flex;
+    flex-direction: column;
+}
```

Comment:
You have a small typo here that is making your calculator not render the proper size, the id of your app component is `app` and not `App`, fix that and it will be of the right size.

Reply:
I changed it. Thanks a lot. :)